app/src/main/java/com/ustwo/boilerplate/feature/helloworld/HelloWorldActivity.java

```diff
@@ -36,7 +36,8 @@ protected int getLayoutId() {
     return this;
   }
 
-  private HelloWorldComponent createComponentAndInject() {
+  @Override
+  protected HelloWorldComponent createComponentAndInject() {
```

Comment:
As we need ApplicationComponent in every activity maybe it'll be useful to create method in BaseActivity to return it?